### DISCUSSION TITLE
typo in link to discussions

### DISCUSSION THREAD

**Discussion by gaby-de-wilde** [eu-digital-identity-wallet/eudi-wallet-reference-implementation-roadmap#50] [General]
https://github.com/eu-digital-identity-wallet/.github/blob/main/profile/reference-implementation.md

Contributions and Feedback > Roadmap suggestions > .... Discussions forum](https://github.com/eu-digital-identity-wallet/eudi-wallet-reference-implementation-roadmap/discussions)). <---- there is an unwanted ")" that ends up glued to the end of the url.

**Comment by paparitsidis-nci** [comment]
Thank you for highlighting this typo. It is now fixed.
